src/test/resources/testResult/wqp/orgData/orgData.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<dataset>
+	<org_data_swap_biodata 
```

Comment:
It would be nice to see data with a second (or third) organization to show the 'dense_rank' is working.